### PR TITLE
refactor: validate config

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- validate yml config using json schema

### CODE REVIEW

File: src/config/schema.ts [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+      type: "object",
+      properties: {
+        LOW_BOUND_RATIO_SHORTING: { type: "number" },
+        LOW_SAFEBOUND_RATIO_SHORTING: { type: "number" },
+        HIGH_SAFEBOUND_RATIO_SHORTING: { type: "number" },
+        HIGH_BOUND_RATIO_SHORTING: { type: "number" },
+        LOW_BOUND_LEVERAGE: { type: "number" },
+        LOW_SAFEBOUND_LEVERAGE: { type: "number" },
+        HIGH_SAFEBOUND_LEVERAGE: { type: "number" },
+        HIGH_BOUND_LEVERAGE: { type: "number" },
```
Comment (dolcalmi) [inline src/config/schema.ts:81]: if these are not env variables should keep camel case standard but... :thinking:  are we using this values? this seems to be more a dealer config than backend config

**Review by dolcalmi — CHANGES_REQUESTED** [review]
looks good.. just a minor comment

File: src/config/schema.ts (hunk 1 above)
Comment (nicolasburtey) [reply src/config/schema.ts:81]: probably not used any more. would probably make sense to look at all the value in the yaml to see if they're still used as part of this PR

Comment (UncleSamtoshi) [reply src/config/schema.ts:81]: It does not appear to be used by the Galoy app, so I will remove it.

**Review by dolcalmi — APPROVED** [review]

**Pull request merged**